### DISCUSSION TITLE
Is this project still being developed?

### DISCUSSION THREAD

**Discussion by DLakin01** [aquasecurity/trivy-operator#2270] [Development]
Wondering if this project is still being actively maintained/improved. The last release was in June, and it looks like recent issues haven't been engaged with, as well as some issues being in the backlog for a long time. I'm a big fan of trivy-operator, but just want to make sure the project is still actively being worked on!

**Comment by amrap030** [comment]
I have the same question, I am also a big fan and it would be great to know if this project will still be maintained? Otherwise I have to look for alternatives as it already doesn't work with Kubernetes Version 1.31 anymore :(

**Comment by wssbck** [comment]
I would like to ask the same. It seems that @chen-keinan is not really active on GitHub these days. Perhaps someone maintaining the main repository of https://github.com/aquasecurity/trivy could shed some light, for example @knqyf263 or @DmitriyLewen?

**Comment by itaysk** [comment]
Hi, Chen who used to passionately maintain this project is no longer with Aqua and we are currently reprioritizing his unattended responsibilities. We will pick up the maintenance gap eventually, but I want to set the expectations that it will not return to the same level of engagement and initiative as before. In the upcoming period our focus will be less about ideation of new features and functionality, and more about stability and quality of this project.

**Reply by wssbck** [reply]
Thanks for the update @itaysk

**Reply by DLakin01** [reply]
@itaysk Thank you for the clarification. Do you know if compatibility with newer versions of Kubernetes will be a priority?

**Reply by lvedder** [reply]
thanks for clarifying, I was also wondering!

**Reply by itaysk** [reply]
> Do you know if compatibility with newer versions of Kubernetes will be a priority?

yes we will make sure that Trivy Operator work on future versions of k8s

**Reply by jemag** [reply]
we really hope that this project keeps maintenance and stability over time. We have invested significant energy in making trivy-operator part of our core setup.

**Reply by cnwaldron** [reply]
@itaysk There are several PRs put in to fix #2251. Is it possible to get this issue prioritized ASAP as it's completely stopping Trivy-operator from being able to be used on newer Kubernetes clusters?

**Reply by jemag** [reply]
I agree, this PR should become a priority. If trivy-operator is not compatible with 1.31.0, teams using it will worry/start looking at alternatives.

**Reply by itaysk** [reply]
thanks for bringing to my attention, yes, we'll prioritize it. tracking in the relevant issue.

**Comment by williamsjo-reninc** [comment]
Do you have an ETA on when these fixes will be officially released?

**Reply by afdesk** [reply]
I hope a new release will be cut in a few days (today/tomorrow)

**Reply by williamsjo-reninc** [reply]
Awesome. Thank you!
